### ISSUE TITLE
About AMC Token failed to add the Arbitrum Network Logo, Contract address: 0x299142a6370e1912156E53fBD4f25D7ba49DdcC5

### ISSUE THREAD

**Issue by AIMetaClub** [TP-Lab/tokens#9241]
Hi, we submitted a request to display the logo of both BSC and Arbitrum network AMC Token in TP a month ago, and now the BSC network one is displayed normally, but the Arbitrum network one is not. Currently on Github, it shows that both BSC network and Arbitrum network commits have been merged. Please check again, thanks.

**Comment by opera0001** [comment]
May I ask what is the order number you submitted to github? I will check it for you, thank you.

**Comment by AIMetaClub** [comment]
Order number is #9078

In this order @opera0001 questioned that the contract on Arbitrum is not open source, in fact it is open source, we can view it from this link: https://arbiscan.io/token/0x299142a6370e1912156e53fbd4f25d7ba49ddcc5#code

In addition, the Arbitrum contract in the order #9078 has also been merged into the code, but @opera0001 said that it needs to be submitted again. Do we need to do this? If submit it again, it is a duplicate submission? 
Thanks.

**Comment by opera0001** [comment]
https://github.com/TP-Lab/tokens/pull/9078   

In this order, you have submitted the tpt donation record only once and have not seen other records. Because you need to submit the application for two logos, you need to submit the tpt for two times.

**Comment by AIMetaClub** [comment]
Thanks, it's clear.

We also encountered another problem. The BSC network that has been submitted will have this kind of safety reminder on the picture. What is the problem?

![image](https://user-images.githubusercontent.com/118162660/233286727-db72f82f-2d79-4813-9f75-1f77b37cdfe8.png)

**Comment by opera0001** [comment]
> Thanks, it's clear.
> 
> We also encountered another problem. The BSC network that has been submitted will have this kind of safety reminder on the picture. What is the problem?
> 
> ![image](https://user-images.githubusercontent.com/118162660/233286727-db72f82f-2d79-4813-9f75-1f77b37cdfe8.png)

This token has yet to start trading now, so temporary state see is like this.

**Comment by opera0001** [comment]
Is this token not being traded yet?When it starts trading, the display here changes.
